UnicornEngine/video/include/unicorn/video/Camera.hpp

```diff
@@ -19,105 +19,100 @@ namespace unicorn
 namespace video
 {
 /**
- * @brief Type of projection, which we can use in Camera
+ * @brief Type of camera projection
  */
 enum class ProjectionType
 {
     Perspective
 };
 
 /**
- * @brief Camera class, which create View and Projection matrices.
+ * @brief Camera class, hold view and projection matrices.
  */
 class Camera
 {
 public:
     /**
      * @brief Camera constuctor
-     * @param postion Default position of camera
+     * @param position Default position of camera
      * @param direction Default view direction of camera
      */
-    UNICORN_EXPORT Camera(glm::vec3 postion, glm::vec3 direction);
+    UNICORN_EXPORT Camera(const glm::vec3& position, const glm::vec3& direction);
 
     /**
-     * @brief 
+     * @brief Sets perspective
      * @param fov Field of view
-     * @param aspect Aspect ratio - 4/3, 16/9 and so on.
-     * @param znear Near camera's frustrum plane
-     * @param zfar Far camera's frustrum plane
+     * @param aspect Aspect ratio
+     * @param znear Camera's near frustrum plane
+     * @param zfar �amera's far frustrum plane
      */
     UNICORN_EXPORT void SetPerspective(float fov, float aspect, float znear, float zfar);
     /**
-     * @brief Setting new aspect ratio
-     * @param aspect New aspect value
+     * @brief Sets aspect ratio
+     * @param aspect Aspect value
      */
-    UNICORN_EXPORT void UpdateAspectRatio(float aspect);
+    UNICORN_EXPORT void SetAspectRatio(float aspect);
     /**
      * @brief Translate camera in space
-     * @param delta vector of translation, for example {15.f, 3.f, 0.f} will translate x by 15m, y by 3 and z by 0
+     * @param delta vector of translation
      */
     UNICORN_EXPORT void Translate(const glm::vec3& delta);
     /**
-     * @brief Setting new direction of camera
+     * @brief Sets direction of camera
      * @param direction New direction value
      */
     UNICORN_EXPORT void SetDirection(const glm::vec3& direction);
     /**
-     * @brief Setting new up vector for camera
+     * @brief Sets up vector for camera
      * @param upVector New up vector value
      */
     UNICORN_EXPORT void SetUpVector(const glm::vec3& upVector);
     /**
-     * @brief Setting new position for camera
+     * @brief Sets position for camera
      * @param position New position value
      */
     UNICORN_EXPORT void SetPosition(const glm::vec3& position);
     /**
-     * @brief Setting new projection type for camera
-     * @param newType New type value
+     * @brief Sets projection type for camera
+     * @param type New type value
      */
-    UNICORN_EXPORT void SetProjectionType(ProjectionType newType);
+    UNICORN_EXPORT void SetProjectionType(ProjectionType type);
```

Comment:
Does it make sense to externally set the projection type without updating projection matrix? `SetPerspective` should take care of projection type.